app/Actions/Network/AllocateNetworkBlock.php

```
@@ -0,0 +1,132 @@
+    }
+
+    /**
+     * @param  Collection<int, Server>  $memberServers
+     * @return array<int, string>
+     */
+    private function memberSubnets(Collection $memberServers): array
+    {
+        return $memberServers
+            ->flatMap(fn (Server $server): Collection => $server->ipAddresses()->get())
+            ->filter(fn ($address): bool => filter_var($address->ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4) !== false)
+            ->map(fn ($address): string => Cidr::canonical($address->ip.'/'.$address->prefix_length))
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\Cidr' in method 'memberSubnets'. (undefined)

(StaticAccess)